pkg/builder/vm.go

```diff
@@ -51,8 +51,9 @@ func NewVMBuilder(creator string) *VMBuilder {
 		Cores: defaultVMCPUCores,
 	}
 	resources := kubevirtv1.ResourceRequirements{
-		Requests: corev1.ResourceList{
+		Limits: corev1.ResourceList{
```

Comment:
Will need to bump the harvester [docker-machine-driver](https://github.com/harvester/docker-machine-driver-harvester/blob/master/go.mod#L11) to leverage the `builder pkg file` changes. cc @futuretea

Reply:
I've opend a WIP PR for that 😆 
https://github.com/harvester/docker-machine-driver-harvester/pull/27